packages/terminal/src/node/shell-process.ts

```diff
@@ -20,9 +20,25 @@ export type ShellProcessFactory = (options: ShellProcessOptions) => ShellProcess
 export const ShellProcessOptions = Symbol("ShellProcessOptions");
 export interface ShellProcessOptions {
     shell?: string,
+    args?: string[],
     rootURI?: string,
     cols?: number,
-    rows?: number
+    rows?: number,
+    env?: { [key: string]: string | null },
+}
+
+function setUpEnvVariables(optionsEnv?:  { [key: string]: string | null }): NodeJS.ProcessEnv {
+    const prEnv: NodeJS.ProcessEnv = {...process.env};
+    if (optionsEnv) {
+        for (const envName of Object.keys(optionsEnv)) {
+            const envValue = optionsEnv[envName];
+            if (envValue) {
```

Comment:
Defining an environment variable to an empty string should work.  Can you verify that it works?  The env var should be visible in the child process, with an empty value.